.github/cloud-nuke/config.yml

```diff
@@ -0,0 +1,2 @@
+s3:
```

Comment:
I think the configuration should include buckets for tests and VPCs created in tests
```
s3:
  include:
    names_regex:
      - "^terragrunt-test-bucket-[a-zA-Z0-9]{6}.*"


```